ocular/src/chain.rs

```diff
@@ -1,4 +1,187 @@
+use std::str::FromStr;
+
+use crate::ChainRegistryError;
+
 pub mod client;
 pub mod config;
 pub mod info;
 pub mod registry;
+
+pub enum Chains {
```

Comment:
I'm wondering if theres a way of populating all these automatically. Maybe if we parsed the chain registry somehow we could get all active chains/chain names? Thoughts?

Reply:
Not sure I understand the use case. The idea behind the enum is to provide the user with knowledge of the available chains in the auto-complete feature of their IDE or in the doc and be used to construct a client. Are you saying auto-generate the rust enum at compile time?

Reply:
Yeah, although I'm not sure how pragmatic or relevant that would be the more I think about it.

Reply:
If it only becomes available at compile time there is no longer a need for the enum since the user will have already written their code at that point.